### PR TITLE
PMM Components and versions

### PR DESCRIPTION
**Pull request by rasika-chivate**

### CODE REVIEW

**Comment by puneet0191** [comment]
@rasika-chivate  who verified the versions of all of these components?

**Comment by rasika-chivate** [comment]
> @rasika-chivate who verified the versions of all of these components?

@BupycHuk I had shared the google doc for the same...

File: docs/details/pmm_components_and_versions.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+| Nginx    | 1.20.1 | [Nginx Documentation](http://nginx.org/en/docs/)|[Github Nginx](https://github.com/nginx/nginx-releases)                                                    |                                          
+| Percona Distribution for PostgreSQL  | 14.1    | [Percona Distribution for PostgreSQL 14 Documentation](https://www.percona.com/doc/postgresql/LATEST/index.html)|              |                                          
+| Clickhouse| 21.3.14.1 |[ClickHouse Documentation Documentation](https://clickhouse.com/docs/en/)|[Github ClickHouse](https://github.com/ClickHouse/ClickHouse)|                                          
+| PerconaToolkit  | 3.4.0    | [Percona Toolkit Documentation](https://www.percona.com/doc/percona-toolkit/3.0/index.html)|[Github Percona Toolkit](https://github.com/percona/percona-toolkit)|                                          
+| Alertmanager  | 0.22.0   | [Alertmanager Documentation](https://prometheus.io/docs/alerting/latest/alertmanager/)|[Github Alertmanager](https://github.com/prometheus/alertmanager)|
```
Comment (denisok) [inline docs/details/pmm_components_and_versions.md:14]: @artemgavrilov please review this version, if it is correct.

File: docs/details/pmm_components_and_versions.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+| Alertmanager  | 0.22.0   | [Alertmanager Documentation](https://prometheus.io/docs/alerting/latest/alertmanager/)|[Github Alertmanager](https://github.com/prometheus/alertmanager)|                                          
+| MongoDB exporter  | 0.34.0    | |[Github percona/mongodb_exporter](https://github.com/percona/mongodb_exporter)|                                          
+| RDS exporter  | 0.7.2    | |[Github percona/rds_exporter](https://github.com/percona/rds_exporter)|                                          
+| Mysqld exporter| V0.14.0 and Percona changes    | [MySQL Server Exporter Documentation](https://grafana.com/oss/prometheus/exporters/mysql-exporter/)|[Github prometheus/mysqld_exporter](github.com/percona/mysqld_exporter)    |                                          
+| Node exporter|v1.3.1 and Percona changes| [Node Exporter Documentation](https://prometheus.io/docs/guides/node-exporter/)|[Github node exporter](https://github.com/percona/node_exporter)    |                                          
+| Azure exporter|       |            |[Github azure_metrics_exporter](https://github.com/percona/azure_metrics_exporter)    |
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply docs/details/pmm_components_and_versions.md:19]: We can use hashes for this kind of build.

**Review by BupycHuk — APPROVED** [review]

**Comment by BupycHuk** [comment]
@puneet0191 I've verified versions.

**Comment by puneet0191** [comment]
thanks, I am fine with this but we need to identify who will maintain this page I doubt this will become outdated very fast

**Review by puneet0191 — APPROVED** [review]

File: docs/details/pmm_components_and_versions.md (hunk 2 above)
Comment (denisok) [inline docs/details/pmm_components_and_versions.md:19]:
```suggestion
| Azure exporter| [61e6677](https://github.com/percona/azure_metrics_exporter/commits/pmm-2.30.0)      |            |[Github azure_metrics_exporter](https://github.com/percona/azure_metrics_exporter)    |                                          
```

**Review by denisok — APPROVED** [review]

File: docs/details/pmm_components_and_versions.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# PMM components and versions
+
+The following table lists all the PMM client/server components and their versions:
+
+
+| Component| Version| URL                                           | GitHub URL                                                                                                  |
```
Comment (atymchuk) [inline docs/details/pmm_components_and_versions.md:6]:
```suggestion
| Component| Version| Documentation                                 | Repository                                                                                                  |
```

Comment (atymchuk) [reply docs/details/pmm_components_and_versions.md:6]: @rasika-chivate It feels like we still call fill in the blank cells in the table. I have a number of suggestions, let's discuss.

Screenshot 2022-09-21 at 10 58 11

File: docs/details/pmm_components_and_versions.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+| MongoDB exporter  | 0.34.0    | |[Github percona/mongodb_exporter](https://github.com/percona/mongodb_exporter)|                                          
+| RDS exporter  | 0.7.2    | |[Github percona/rds_exporter](https://github.com/percona/rds_exporter)|                                          
+| Mysqld exporter| V0.14.0 and Percona changes    | [MySQL Server Exporter Documentation](https://grafana.com/oss/prometheus/exporters/mysql-exporter/)|[Github prometheus/mysqld_exporter](github.com/percona/mysqld_exporter)    |                                          
+| Node exporter|v1.3.1 and Percona changes| [Node Exporter Documentation](https://prometheus.io/docs/guides/node-exporter/)|[Github node exporter](https://github.com/percona/node_exporter)    |                                          
+| Azure exporter| [61e6677](https://github.com/percona/azure_metrics_exporter/commits/pmm-2.30.0)      |            | [Github azure_metrics_exporter](https://github.com/percona/azure_metrics_exporter)    |
```
[the remark this answers is not in the document]
Comment (denisok) [reply docs/details/pmm_components_and_versions.md:19]: but we don't have that version. we do have branch, but we also need to start tags. So ppl could for sure know what exactly is in the release.

[the remark this answers is not in the document]
Comment (atymchuk) [reply docs/details/pmm_components_and_versions.md:19]: Sorry, I mentioned version, but in reality I meant the branch, hence my commit suggestion above.

I agree, tags would work best )

**Review by atymchuk — APPROVED** [review]

File: docs/details/pmm_components_and_versions.md (hunk 1 above)
Comment (artemgavrilov) [reply docs/details/pmm_components_and_versions.md:14]: All good here

File: docs/details/pmm_components_and_versions.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+
+| Component| Version| Documentation                                 | Repository                                                                                                  |
+|----------|--------|---------------------------------------------- |-------------------------------------------------------------------------------------------------------------|
+| Grafana  | 9.1    | [Grafana Documentation](https://grafana.com/docs/grafana/latest/)|[Github Grafana](https://github.com/percona-platform/grafana)|                                          
+| VictoriaMetrics| 1.77.1    | [VictoriaMetrics Documentation](https://docs.victoriametrics.com/)|[Github VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics)    |                                          
+| Nginx    | 1.20.1 | [Nginx Documentation](http://nginx.org/en/docs/)|[Github Nginx](https://github.com/nginx/nginx-releases)                                                    |                                          
+| Percona Distribution for PostgreSQL  | 14.1    | [Percona Distribution for PostgreSQL 14 Documentation](https://www.percona.com/doc/postgresql/LATEST/index.html)|              |
```
Comment (artemgavrilov) [inline docs/details/pmm_components_and_versions.md:11]:
```suggestion
| Percona Distribution for PostgreSQL  | 14.5    | [Percona Distribution for PostgreSQL 14 Documentation](https://www.percona.com/doc/postgresql/LATEST/index.html)|              |                                          
```

File: docs/details/pmm_components_and_versions.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+| Grafana  | 9.1    | [Grafana Documentation](https://grafana.com/docs/grafana/latest/)|[Github Grafana](https://github.com/percona-platform/grafana)|                                          
+| VictoriaMetrics| 1.77.1    | [VictoriaMetrics Documentation](https://docs.victoriametrics.com/)|[Github VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics)    |                                          
+| Nginx    | 1.20.1 | [Nginx Documentation](http://nginx.org/en/docs/)|[Github Nginx](https://github.com/nginx/nginx-releases)                                                    |                                          
+| Percona Distribution for PostgreSQL  | 14.1    | [Percona Distribution for PostgreSQL 14 Documentation](https://www.percona.com/doc/postgresql/LATEST/index.html)|              |                                          
+| Clickhouse| 21.3.14.1 |[ClickHouse Documentation Documentation](https://clickhouse.com/docs/en/)|[Github ClickHouse](https://github.com/ClickHouse/ClickHouse)|
```
Comment (artemgavrilov) [inline docs/details/pmm_components_and_versions.md:12]:
```suggestion
| Clickhouse| 21.3.20.1 |[ClickHouse Documentation Documentation](https://clickhouse.com/docs/en/)|[Github ClickHouse](https://github.com/ClickHouse/ClickHouse)|                                          
```

Comment (artemgavrilov) [reply docs/details/pmm_components_and_versions.md:12]: @BupycHuk I double checked versions in dev-latest. It seems that we should update couple records here.

**Review by artemgavrilov — APPROVED** [review]

File: docs/details/pmm_components_and_versions.md (hunk 6 above)
Comment (BupycHuk) [reply docs/details/pmm_components_and_versions.md]: Great, looks like I missed it

**Pull request merged**